src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  --color-col: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  color: var(--color-col);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  position: relative;
```

Comment:
```suggestion
  list-style: none;
  margin: 0;
  padding: 0;
  display: flex;
  color: var(--color-col);
  font-size: 12px;
  font-weight: 500;
  text-transform: uppercase;
```

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3746/files#r13077330
still not fixed